### ISSUE TITLE
Conditional include of socialaccount related snippets in login.html fails when socialaccount not installed.

### ISSUE THREAD

**Issue by chrisbura** [pennersr/django-allauth#459]
The following section in `accounts/login.html` will error out if `allauth.socialaccount` is not installed and `TEMPLATE_DEBUG = True` since it can't find the socialaccount templatetags. It will fail silently when `TEMPLATE_DEBUG = False` and display as expected.

```
{% if socialaccount.providers %}
...
    {% include "socialaccount/snippets/provider_list.html" with process="login" %}
...
{% include "socialaccount/snippets/login_extra.html" %}
...
{% endif %}
```

I did some research and it seems to be the way that django parses the template. It parses the `include` even if it will never actually be rendered due to the `if` statement.

See 
https://code.djangoproject.com/ticket/7931
https://groups.google.com/forum/#!topic/django-users/oq1UM5yDNKM

This occurs on Django 1.6

I just tested it on the dev version and it seems to work fine. Might be because 1.7 got some new features with respect to the `include` tag.

https://docs.djangoproject.com/en/dev/releases/1.7/

**Comment by derek-adair** [comment]
Ok after much meditation on this; i think this is just *can* be reduced to an issue of template inheritance but may justify a deeper refactor.

The decision is to either; 
a) remove the reference to the app `allauth.socialaccount` and **let** `allauth.templates.socialaccount.login` **be a stand alone login template** that inherits directly from `allauth.templates.account.login.html`.  Users who want to use the official social auth templates can just do so explicitly.

**or** 

b) just admit defeat and combine the apps.  As a user I would not care if the socialaccount app where included while the individual provider apps where still separate.

-------------------

Each has its own pros/cons.  Not really sure which would be more in the spirit of this project.  It seems like social login is a primary feature so that is worth considering just smashing these two apps together in the same namespace.  It honestly _seems_ like combining the namespace would be less invasive to the end user.

**Comment by pennersr** [comment]
Fixed via 0c4816b8

**Comment by pennersr** [comment]
https://codeberg.org/allauth/django-allauth/issues/459
